etcd/containers/0.13.1/etcd/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+ACCT=${ACCT:-rancher}
```

Comment:
Just to be in sync with other microservices, can you please use `REPO` instead of `ACCT`?